src/constants/Colors.js

```diff
@@ -0,0 +1,14 @@
+//  Light mode Colors
```

Comment:
What about creating themes instead, something like:

```jsx
const themes = {
  dark: {
    primary: ...
  },
  light: {
    primary: ...
  }
}
```

Then you can re-use this them among all your components and styled components